web/packages/teleport/src/services/localStorage/localStorage.ts

```diff
@@ -161,6 +162,12 @@ const storage = {
     window.localStorage.removeItem(KeysEnum.THEME);
   },
 
+  areUnifiedResourcesEnabled(): boolean {
+    return (
+      window.localStorage.getItem(KeysEnum.UNIFIED_RESOURCES_ENABLED) === 'true'
```

Comment:
i'm not sure i'm seeing where this value gets set? (how do we get the true value?)